internal/cmd/operator-sdk/scorecard/cmd.go

```diff
@@ -198,8 +198,14 @@ func (c *scorecardCmd) run() (err error) {
 	if c.list {
 		scorecardTests = o.List()
 	} else {
+		runnerSA := ""
+		if o.Config.ServiceAccount != "" {
+			runnerSA = o.Config.ServiceAccount
+		} else {
+			runnerSA = c.serviceAccount
+		}
```

Comment:
Since you're already initializing it, might as well set it to the default value

```suggestion
		runnerSA := c.serviceAccount
		if o.Config.ServiceAccount != "" {
			runnerSA = o.Config.ServiceAccount
		}
```